resources/views/editUserProfile.blade.php

```diff
@@ -0,0 +1,109 @@
+
+@extends('layouts.app')
+
+@section('title', 'John Doe')
+
+@section('content')
+<div class="padded content">
+	<h1>John Doe</h1>
+	<div class="ui form error">
+		<!-- User image -->
+		<div id ="userImage" class="field">
+			<label id="userImage" for="userImage">Photo</label>
+			<div class="userImageUploaderContainer">
+				<div class="imagePreviewContainer">
+					<img id="userImagePreview" src="https://lorempixel.com/400/400" alt="User image" class="ui small circular image preview"/>
+				</div>
+				<a href="#" class="imgUploader">
+					<button id="uploadImage" class="ui button primary">Upload image</button>
+				</a>
+			</div>
+			<input id="imgInput" type="file" name="userImage" style="display:none" accept="image/x-png,image/jpeg">
+		</div>
+		<!-- User name -->
+		<div id ="userName" class="field">
+			<label id="userName" for="userName">Name</label>
+			<input type="text" name="userName" placeholder="John Doe" value="Katie Arriaga">
```

Comment:
Katie, aquí me refería a que el `for` del `label` debe empatar con el `id` del `input`. El label no necesita un `id` (salvo que lo requieras para algo especial). Una manera fácil de verificar que funciona es si le picas al label debe darle el focus al `input` correspondiente.